src/System.Collections.NonGeneric/tests/CollectionBaseTests.cs

```diff
@@ -242,8 +242,10 @@ public static void CopyTo_Invalid()
         {
             MyCollection collBase = CreateCollection(100);
             var fooArr = new Foo[100];
-            Assert.Throws<ArgumentOutOfRangeException>("dstIndex", () => collBase.CopyTo(fooArr, -1)); // Index < 0
-            Assert.Throws<ArgumentException>(string.Empty, () => collBase.CopyTo(fooArr, 50)); // Index + fooArray.Length > collBase.Count
+            // Index < 0
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("destinationIndex", "dstIndex", () => collBase.CopyTo(fooArr, -1));
+            // Index + fooArray.Length > collBase.Count
+            AssertExtensions.Throws<ArgumentException>("destinationArray", string.Empty, () => collBase.CopyTo(fooArr, 50));
```

Comment:
I used `string.Empty` everywhere except in ArrayTests.cs. Within that file there were already 68 occurrences of `""` and none of `string.Empty` so I kept everything like the former.